### PR TITLE
[hi] Added  content/hi/docs/concepts/architecture/nodes.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
Issue Number #39548

Localized en/docs/concepts/architecture/nodes.md in hindi language :)

### CODE REVIEW

File: content/hi/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+- caesarxuchao
+- dchen1107
+title: नोड्स
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes _Nodes_ पर चलने के लिए पॉड्स में कंटेनर रखकर आपका {{< glosary_tooltip text="workload" term_id="workload" >}} चलाता है।
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:12]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+क्लस्टर के आधार पर एक नोड वर्चुअल या भौतिक मशीन हो सकता है। प्रत्येक नोड
+द्वारा प्रबंधित किया जाता है
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+और चलाने के लिए आवश्यक सेवाएं शामिल हैं
+{{< glosary_tooltip text="Pods" term_id="pod" >}}.
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:17]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="पॉड्स" term_id="pod" >}}.
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+नोड पर [घटक](/docs/concepts/overview/components/#node-components) में शामिल हैं
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}},
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}, और
+{{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}.
+
+<!-- body -->
+
+## प्रबंध
+
+{{< glosary_tooltip text="API server" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:31]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="API सर्वर" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 4]
```diff
@@ -0,0 +1,649 @@
...
+कि एक क्यूबलेट एपीआई सर्वर पर पंजीकृत है जो `metadata.name` से मेल खाता है
+नोड का क्षेत्र। यदि नोड स्वस्थ है (यानी सभी आवश्यक सेवाएं चल रही हैं),
+तो यह पॉड चलाने के योग्य है। अन्यथा, उस नोड को किसी भी क्लस्टर गतिविधि के लिए अनदेखा कर दिया जाता है
+जब तक यह स्वस्थ न हो जाए।
+
+{{< नोट >}}
+कुबेरनेट्स ऑब्जेक्ट को अमान्य नोड के लिए रखता है और यह देखने के लिए जांच जारी रखता है कि क्या
+यह स्वस्थ हो जाता है।
+
+आपको, या किसी {{< glosary_tooltip term_id="controller" text="controller">}} को स्पष्ट रूप से
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:63]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 5]
```diff
@@ -0,0 +1,649 @@
...
+
+### नोड्स का स्व-पंजीकरण
+
+जब क्यूबलेट फ्लैग `--register-node` ट्रू (डिफ़ॉल्ट) होता है, तो क्यूबलेट प्रयास करेगा
+एपीआई सर्वर के साथ खुद को पंजीकृत करें। यह पसंदीदा पैटर्न है, जिसका उपयोग अधिकांश डिस्ट्रोज़ द्वारा किया जाता है।
+
+स्व-पंजीकरण के लिए, क्यूबलेट निम्नलिखित विकल्पों के साथ शुरू होता है:
+
+- `--kubeconfig` - API सर्वर पर स्वयं को प्रमाणित करने के लिए क्रेडेंशियल्स का पथ।
+- `--cloud-provider` - {{< glosary_tooltip text="क्लाउड प्रोवाइडर" टर्म_आईडी="क्लाउड-प्रोवाइडर" >}} से कैसे बात करें अपने बारे में मेटाडेटा पढ़ने के लिए।
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:89]: Typo in the `glossary_tooltip` shortcode name. Also, the localization of the shortcode in this line looks incorrect.

```suggestion
- `--cloud-provider` - {{< glossary_tooltip text="क्लाउड प्रदाता" term-id="cloud-provider" >}} से कैसे बात करें अपने बारे में मेटाडेटा पढ़ने के लिए।
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 6]
```diff
@@ -0,0 +1,649 @@
...
+- `--register-node` - स्वचालित रूप से एपीआई सर्वर के साथ पंजीकरण करें।
+- `--register-with-taints` - दी गई सूची के साथ नोड को पंजीकृत करें
+  {{< glosary_tooltip text="taints" term_id="taint" >}} (अल्पविराम से अलग `<कुंजी>=<मान>:<प्रभाव>`)।
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:92]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 7]
```diff
@@ -0,0 +1,649 @@
...
+पॉड को नोड को सौंपे गए नए लेबल के खिलाफ दागी जा सकता है, जबकि अन्य
+पॉड्स, जो उस पॉड के साथ असंगत हैं, इस नए के आधार पर शेड्यूल किए जाएंगे
+लेबल। नोड का पुनः पंजीकरण सुनिश्चित करता है कि सभी पॉड्स खाली और ठीक से हो जाएंगे
+पुनर्निर्धारित।
+{{< /नोट >}}
+
+### मैनुअल नोड प्रशासन
+
+आप नोड ऑब्जेक्ट्स का उपयोग करके बना और संशोधित कर सकते हैं
+{{< glosary_tooltip text="kubectl" term_id="kubectl" >}}.
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:121]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="क्यूबकण्ट्रोल" term_id="kubectl" >}}.
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 8]
```diff
@@ -0,0 +1,649 @@
...
+    "message": "kubelet is posting ready status",
+    "lastHeartbeatTime": "2019-06-05T18:38:35Z",
+    "lastTransitionTime": "2019-06-05T11:41:27Z"
+  }
+]
+```
+
+यदि तैयार स्थिति की `स्थिति` लंबे समय तक `अज्ञात` या `गलत` बनी रहती है
+`पॉड-एविक्शन-टाइमआउट` की तुलना में (एक तर्क पास किया गया
+{{< glosary_tooltip text="kube-controller-manager" term_id="kube-controller-manager"
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:216]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 9]
```diff
@@ -0,0 +1,649 @@
...
+>}}), फिर [नोड नियंत्रक](#नोड-नियंत्रक) ट्रिगर करता है
+{{< glosary_tooltip text="API-initiated eviction" term_id="api-eviction" >}}
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:218]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 10]
```diff
@@ -0,0 +1,649 @@
...
+`टर्मिनेटिंग` या `अज्ञात` स्थिति में होना। ऐसे मामलों में जहां कुबेरनेट्स से निष्कर्ष नहीं निकाला जा सकता है
+अंतर्निहित आधारभूत संरचना यदि कोई नोड स्थायी रूप से क्लस्टर छोड़ देता है, तो क्लस्टर व्यवस्थापक
+नोड ऑब्जेक्ट को हाथ से हटाने की आवश्यकता हो सकती है। कुबेरनेट्स से नोड वस्तु को हटाने का कारण बनता है
+नोड पर चल रहे सभी पॉड ऑब्जेक्ट्स को एपीआई सर्वर से हटा दिया जाएगा और उनके नाम को मुक्त कर दिया जाएगा।
+
+जब नोड्स पर समस्याएँ होती हैं, तो कुबेरनेट्स नियंत्रण विमान स्वचालित रूप से बनाता है
+[दाद](/docs/concepts/scheduling-eviction/taint-and-toleration/) जो शर्तों से मेल खाते हैं
+नोड को प्रभावित करना।
+नोड को पॉड असाइन करते समय शेड्यूलर नोड के दोष को ध्यान में रखता है।
+पॉड में {{< glosary_tooltip text="tolerations" term_id="toleration" >}} भी हो सकता है जो
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:237]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 11]
```diff
@@ -0,0 +1,649 @@
...
+
+कुबेरनेट्स नोड्स द्वारा भेजी गई दिल की धड़कन, आपके क्लस्टर को यह निर्धारित करने में मदद करती है
+प्रत्येक नोड की उपलब्धता, और विफलताओं का पता चलने पर कार्रवाई करना।
+
+नोड्स के लिए दिल की धड़कन के दो रूप होते हैं:
+
+* नोड के `.status` के अपडेट
+* [लीज](/docs/concepts/architecture/leases/) ऑब्जेक्ट्स
+  'क्यूब-नोड-लीज' के भीतर
+  {{< glosary_tooltip term_id="namespace" text="namespace">}}.
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:273]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 12]
```diff
@@ -0,0 +1,649 @@
...
+`--node-eviction-rate` की सामान्य दर। कोने का मामला तब होता है जब सभी जोन होते हैं
+पूरी तरह से अस्वस्थ (क्लस्टर में कोई भी नोड स्वस्थ नहीं है)। ऐसे में
+मामला, नोड नियंत्रक मानता है कि कनेक्टिविटी के साथ कुछ समस्या है
+नियंत्रण तल और नोड्स के बीच, और कोई निष्कासन नहीं करता है।
+(यदि कोई आउटेज हो गया है और कुछ नोड्स फिर से दिखाई देते हैं, तो नोड नियंत्रक करता है
+पॉड्स को शेष नोड्स से बेदखल करें जो अस्वास्थ्यकर या अगम्य हैं)।
+
+नोड कंट्रोलर नोड्स पर चल रहे पॉड को बेदखल करने के लिए भी जिम्मेदार है
+`NoExecute` दागी, जब तक कि उन पॉड्स उस कलंक को बर्दाश्त नहीं करते।
+नोड नियंत्रक {{< glosary_tooltip text="taints" term_id="taint" >}} भी जोड़ता है
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:357]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 13]
```diff
@@ -0,0 +1,649 @@
...
+
+## संसाधन क्षमता ट्रैकिंग {#नोड-क्षमता}
+
+नोड ऑब्जेक्ट नोड की संसाधन क्षमता के बारे में जानकारी ट्रैक करते हैं: उदाहरण के लिए, राशि
+उपलब्ध मेमोरी की संख्या और CPU की संख्या।
+नोड्स जो [स्वयं पंजीकरण](#स्व-पंजीकरण-ऑफ-नोड्स) के दौरान अपनी क्षमता की रिपोर्ट करते हैं
+पंजीकरण। यदि आप [मैन्युअल रूप से](#मैन्युअल-नोड-प्रशासन) एक नोड जोड़ते हैं, तो
+जब आप इसे जोड़ते हैं तो आपको नोड की क्षमता की जानकारी सेट करने की आवश्यकता होती है।
+
+कुबेरनेट्स {{< glosary_tooltip text="scheduler" term_id="kube-scheduler" >}} यह सुनिश्चित करता है
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:369]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 14]
```diff
@@ -0,0 +1,649 @@
...
+{{< feature-state state="beta" for_k8s_version="v1.26" >}}
+
+क्यूबलेट के नोड शटडाउन मैनेजर द्वारा नोड शटडाउन कार्रवाई का पता नहीं लगाया जा सकता है,
+या तो क्योंकि कमांड द्वारा उपयोग किए जाने वाले इनहिबिटर लॉक मैकेनिज्म को ट्रिगर नहीं करता है
+क्यूबलेट या उपयोगकर्ता त्रुटि के कारण, यानी शटडाउनग्रेसपीरियोड और
+ShutdownGracePeriodCriticalPods ठीक से कॉन्फ़िगर नहीं किए गए हैं। कृपया ऊपर देखें
+अनुभाग [ग्रेसफुल नोड शटडाउन](#ग्रेसफुल-नोड-शटडाउन) अधिक विवरण के लिए।
+
+जब एक नोड बंद हो जाता है लेकिन क्यूबलेट के नोड शटडाउन मैनेजर द्वारा पता नहीं लगाया जाता है, तो pods
+जो कि {{< glosary_tooltip text="StatefulSet" term_id="statefulset" >}} का हिस्सा हैं, समाप्त होने की स्थिति में अटक जाएंगे
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:559]: Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
जो कि {{< glosary_tooltip text="स्टेटफुलसेट" term_id="statefulset" >}} का हिस्सा हैं, समाप्त होने की स्थिति में अटक जाएंगे
```

**Comment by Ritikaa96** [comment]
Hi @akash-kumar-saw , please recommit with given suggestions from reviewer. The tests are failing right now, please recommit and trigger the tests.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 15]
```diff
@@ -0,0 +1,649 @@
...
+कि एक क्यूबलेट एपीआई सर्वर पर पंजीकृत है जो `metadata.name` से मेल खाता है
+नोड का क्षेत्र। यदि नोड स्वस्थ है (यानी सभी आवश्यक सेवाएं चल रही हैं),
+तो यह पॉड चलाने के योग्य है। अन्यथा, उस नोड को किसी भी क्लस्टर गतिविधि के लिए अनदेखा कर दिया जाता है
+जब तक यह स्वस्थ न हो जाए।
+
+{{< note >}}
+कुबेरनेट्स ऑब्जेक्ट को अमान्य नोड के लिए रखता है और यह देखने के लिए जांच जारी रखता है कि क्या
+यह स्वस्थ हो जाता है।
+
+आपको, या किसी {{< glossary_tooltip term_id="कंट्रोलर" text="controller">}} को स्पष्ट रूप से
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:63]: The values of `term_id` and `text` parameters seem to be swapped. Please ensure that term_id is in English and text is in Hindi.

```suggestion
आपको, या किसी {{< glossary_tooltip term_id="controller" text="कंट्रोलर">}} को स्पष्ट रूप से
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 16]
```diff
@@ -0,0 +1,649 @@
...
+
+कुबेरनेट्स नोड्स द्वारा भेजी गई दिल की धड़कन, आपके क्लस्टर को यह निर्धारित करने में मदद करती है
+प्रत्येक नोड की उपलब्धता, और विफलताओं का पता चलने पर कार्रवाई करना।
+
+नोड्स के लिए दिल की धड़कन के दो रूप होते हैं:
+
+* नोड के `.status` के अपडेट
+* [लीज](/docs/concepts/architecture/leases/) ऑब्जेक्ट्स
+  'क्यूब-नोड-लीज' के भीतर
+  {{< glossary_tooltip term_id="नेमस्पेस" text="namespace">}}.
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:273]: The values of `term_id` and `text` parameters seem to be swapped. Please ensure that term_id is in English and text is in Hindi.

```suggestion
  {{< glossary_tooltip term_id="namespace" text="नेमस्पेस">}}.
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 17]
```diff
@@ -0,0 +1,649 @@
+ ---
+reviewers:
+- caesarxuchao
+- dchen1107
```
Comment (aj11anuj) [inline content/hi/docs/concepts/architecture/nodes.md:4]:
```suggestion
```

File: content/hi/docs/concepts/architecture/nodes.md [hunk 18]
```diff
@@ -0,0 +1,649 @@
...
+`शर्तें` फ़ील्ड सभी `चल रहे` नोड्स की स्थिति का वर्णन करता है। शर्तों के उदाहरणों में शामिल हैं:
+
+{{< table caption = "Node conditions, and a description of when each condition applies." >}}
+| Node Condition       | Description |
+|----------------------|-------------|
+| `Ready`              | `True` if the node is healthy and ready to accept pods, `False` if the node is not healthy and is not accepting pods, and `Unknown` if the node controller has not heard from the node in the last `node-monitor-grace-period` (default is 40 seconds) |
+| `DiskPressure`       | `True` if pressure exists on the disk size—that is, if the disk capacity is low; otherwise `False` |
+| `MemoryPressure`     | `True` if pressure exists on the node memory—that is, if the node memory is low; otherwise `False` |
+| `PIDPressure`        | `True` if pressure exists on the processes—that is, if there are too many processes on the node; otherwise `False` |
+| `NetworkUnavailable` | `True` if the network for the node is not correctly configured, otherwise `False` |
```
Comment (aj11anuj) [inline content/hi/docs/concepts/architecture/nodes.md:189]: Localize description of table too.

File: content/hi/docs/concepts/architecture/nodes.md [hunk 19]
```diff
@@ -0,0 +1,649 @@
+ ---
+- reviewers:
+- - caesarxuchao
+- - dchen1107
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/nodes.md:4]: These front matter lines contain an error leading to [build failure](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65a983d13d5c3e00083c657a). However, it's worth noting that we should exclude these reviewer lines from this localized version.

```suggestion
```

**Comment by divya-mohan0209** [comment]
@akash-kumar-saw : Checking in if you have the bandwidth to address the build failures:)

**Comment by divya-mohan0209** [comment]
@akash-kumar-saw Since there has been no response/activity on this PR for several months, we'll be closing it. If you have the bandwidth to continue work on this PR, please feel free to reopen the PR and let us know.
/close

**Pull request closed without merging**